backend/routes/sequence.js

```diff
@@ -298,70 +235,109 @@ router.patch("/revision/:revId/sequences/:sequenceId", middlewares.hasUploadAcce
+ * @apiExample {get} Example usage
+ * GET /acme/00000000-0000-0000-0000-000000000000/sequences/00000000-0000-0000-0000-000000000002/legend HTTP/1.1
  *
  * @apiSuccessExample {json} Success-Response
  * HTTP/1.1 200 OK
  * {
  * 	  "Building works": "#aabbcc"
  * 	  "Temporary works": "#ffffff66"
  * }
- *
  */
-router.get("/revision/master/head/sequences/:sequenceId/legend", middlewares.issue.canView, getLegend);
-router.get("/revision/:revId/sequences/:sequenceId/legend", middlewares.issue.canView, getLegend);
+router.get("/sequences/:sequenceId/legend", middlewares.issue.canView, getLegend);
 
 /**
- * @api {delete} /:teamspace/:model/revision(/master/head/|/:revId)/sequences/:sequenceID/legend Delete legend
+ * @api {delete} /:teamspace/:model/sequences/:sequenceID/legend Delete legend
  * @apiName deleteLegend
  * @apiGroup Sequences
  * @apiDescription Delete the legend associated to this sequence
  *
  * @apiUse Sequences
  *
- * @apiExample {delete} Example usage (/master/head)
- * DELETE /acme/00000000-0000-0000-0000-000000000000/revision/master/head/sequences/00000000-0000-0000-0000-000000000002/legend HTTP/1.1
+ * @apiExample {delete} Example usage
+ * DELETE /acme/00000000-0000-0000-0000-000000000000/sequences/00000000-0000-0000-0000-000000000002/legend HTTP/1.1
+ *
+ * @apiSuccessExample {json} Success-Response
+ * HTTP/1.1 200 OK
+ * {}
+ */
+router.delete("/sequences/:sequenceId/legend", middlewares.hasUploadAccessToModel, deleteLegend);
+
+/**
+ * @api {delete} /:teamspace/:model/sequences/:sequenceID Delete sequence
+ * @apiName deleteSequence
+ * @apiGroup Sequences
+ * @apiDescription Delete the custom sequence by ID
  *
- * @apiExample {delete} Example usage (/:revId)
- * DELETE /acme/00000000-0000-0000-0000-000000000000/revision/00000000-0000-0000-0000-000000000001/sequences/00000000-0000-0000-0000-000000000002/legend HTTP/1.1
+ * @apiUse Sequences
+ *
+ * @apiExample {delete} Example usage
+ * DELETE /acme/00000000-0000-0000-0000-000000000000/sequences/00000000-0000-0000-0000-000000000002 HTTP/1.1
  *
  * @apiSuccessExample {json} Success-Response
  * HTTP/1.1 200 OK
  * {}
  */
+router.delete("/sequences/:sequenceId", middlewares.issue.canCreate, deleteSequence);
 
-router.delete("/revision/master/head/sequences/:sequenceId/legend", middlewares.hasUploadAccessToModel, deleteLegend);
-router.delete("/revision/:revId/sequences/:sequenceId/legend", middlewares.hasUploadAccessToModel, deleteLegend);
+/**
+ * @api {post} /:teamspace/:model/sequences Create custom sequence
+ * @apiName createSequence
+ * @apiGroup Sequences
+ * @apiDescription Create custom sequence for model.
+ *
+ * @apiUse Sequences
+ *
+ * @apiExample {post} Example usage
+ * POST /acme/00000000-0000-0000-0000-000000000000/sequences HTTP/1.1
+ * {
+ * 	"name":"Custom Sequence 1",
+ * 	"frames":[
+ * 		{
+ * 			"dateTime":1244246400000,
+ * 			"state":"00000000-0000-0000-0001-000000000002"
```

Comment:
This is synchro state, we should be showing example with viewpoint

Reply:
@Charence  can we make a better example ? the user who go into this document would probably be more interested in things like colour overrides, hiding objects, let's make sure the example includes that